collector/terra/fcd/permanent_store.go

```diff
@@ -0,0 +1,54 @@
+package fcd
+
+import (
+	"github.com/dezswap/cosmwasm-etl/pkg/db/schemas"
+	"github.com/pkg/errors"
+	"gorm.io/gorm"
+)
+
+type permanentStoreImpl struct {
+	db *gorm.DB
+}
+
+var _ permanentStore = (*permanentStoreImpl)(nil)
+
+func NewPermanentStore(db *gorm.DB) permanentStore {
+	return &permanentStoreImpl{db}
+}
+
+// FirstTxOf implements Columbus4Repository.
+func (p *permanentStoreImpl) FirstTxOf(addr string) (schemas.FcdTxLog, error) {
+	tx := schemas.FcdTxLog{}
+	if err := p.db.Where("address = ?", addr).Order("fcd_offset").Limit(1).Find(&tx).Error; err != nil {
+		return schemas.FcdTxLog{}, errors.Wrap(err, "col4RepoImpl.LatestTxOf")
```

Comment:
could you update wrapping message in this file?

Reply:
https://github.com/dezswap/cosmwasm-etl/pull/15/commits/c07da70eaeb89d08fa1de79b63e592ff1769b041